File: File_Handling_.java/CreateFile.c
#include<stdio.h>
#include<stdlib.h>
#include<fcntl.h>

int main()
{
    char Fname[20];
    int fd = 0;

    printf("please enter file name that you want to create\n");
    scanf("%s",Fname);

    fd = creat(Fname,0777);
    if(fd == -1)
    {
        printf("uable to create file\n");
    }
    else
    {
        printf("file is sucessfully created\n");
    }
    return 0;
}

File: File_Handling_.java/CreateFile.java
import java.io.*;


class CreateFile
{
    public static void main(String arg[]) throws Exception
    {
        File fobj = new File("Demo.txt");

        if(fobj.createNewFile())
        {
            System.out.println("file created sucessfully");
        }
        else
        {
            System.out.println("Uable to create the file");
        }
    }
}